pkg/upgrade/templates/v1/cstor_pool.go

```diff
@@ -17,6 +17,25 @@ limitations under the License.
 package templates
 
 var (
+	// CSPPatch is used for patching cstor pool
+	CSPPatch = `{
+    "metadata": {
+        "labels": {
+            "openebs.io/version": "{{.UpgradeVersion}}"
+        }
+    },{{if lt .CurrentVersion "1.8.0"}}
+    "spec": {
+        "poolSpec": {
+            "roThresholdLimit": 85
```

Comment:
Why are we setting it to 85 ? Is it a default value ? 
If yes -- is the code not defaulting.. 
Should be fine -- just trying to understand...

Reply:
yes it is the default value if not mentioned. refer: https://github.com/openebs/maya/pull/1623

Reply:
thanks